cdc/sink/manager_test.go

```diff
@@ -188,7 +190,7 @@ func (s *managerSuite) TestManagerAddRemoveTable(c *check.C) {
 				c.Assert(err, check.IsNil)
 			}
 			_, err := sink.FlushRowChangedEvents(ctx, sink.(*tableSink).tableID, resolvedTs)
```

Comment:
How about change `FlushRowChangedEvents` to following

```go
type Sink interface {
	FlushRowChangedEvents(ctx context.Context, tableID model.TableID, resolvedTs uint64) (uint64, bool, error)
}
```

So we can use the second return value `bool` to determine whether the sink is block or not.

I think it's more intuitive than comparing `ErrFlushTsBlocking`.